pkg/controller/integrationkit/integrationkit_controller.go

```diff
@@ -148,6 +148,11 @@ func add(_ context.Context, mgr manager.Manager, r reconcile.Reconciler) error {
 				}
 
 				for _, kit := range list.Items {
+					if v, ok := kit.Annotations[v1.PlatformSelectorAnnotation]; ok && v != p.Name {
+						// integration kit is waiting for another platform - skip it now
```

Comment:
Would it make sense to log it by any chance?

Reply:
+1

Reply:
Done